pkgs/development/libraries/java/junixsocket/default.nix

```diff
@@ -1,42 +1,44 @@
-{ lib, stdenv, fetchurl, ant, jdk, junit }:
-
-stdenv.mkDerivation rec {
-  name = "junixsocket-1.3";
-
+{ lib, stdenv, fetchurl, jre, runtimeShell }:
+let
+  pname = "junixsocket";
+  version = "2.3.4";
```

Comment:
```suggestion
{ lib, stdenv, fetchurl, jre, runtimeShell }:

let
```

Reply:
Having those in the let block is a personal preference for readability.

Aside from that: You're right, but this is still WIP (I'm gonna give in another try with maven, just wanted to have the version bump first.)